src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -290,6 +293,20 @@ public function setRow($column, array $row)
         return $this;
     }
 
+    public function setHeaderTitle(?string $title): self
+    {
+        $this->headerTitle = $title;
+
+        return $this;
+    }
+
+    public function setfooterTitle(?string $title): self
```

Comment:
method should be renamed to `setFooterTitle`

Reply:
Done